pkg/cloudprovider/cloudprovider.go

```diff
@@ -40,6 +41,7 @@ import (
 	"knative.dev/pkg/logging"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	cloudproviderevents "github.com/aws/karpenter/pkg/cloudprovider/events"
```

Comment:
I've changed my mind a bit about the utility of this being in a separate package. What do you think about collapsing the events package into the same package and suffixing the function call with `Event`?

Reply:
I'd like to follow the semantic of how this is already done, and then opt to re-factor it in a separate PR, if that's alright with you. Why have you changed your mind?

Reply:
> Why have you changed your mind

Creates extra packages for almost no reason. We've already started moving to this semantic in other PRs. Fair point though, I'm fine taking this separately.